### DISCUSSION TITLE
Block app while data roaming

### DISCUSSION THREAD

**Discussion by deimi** [celzero/rethink-app#1203] [Ideas]
What I like in Netguard is the possibility to block apps from accessing the internet while roaming. Unfortunately this feature is missing here, cause I can only block mobile networks in general.
Is there a technical reason for this or is it just not implemented yet? Or was this feature even rejected on purpose.

Just asking before I start thinking about contributing such a feature here.

**Comment by ignoramous** [comment]
You can contribute it, but it won't be straight-forward. I've raised an issue here: https://github.com/celzero/rethink-app/issues/1219
